.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Yarn Install
         run: yarn install --ignore-engines
       - name: Run Mocha Tests
-        run: yarn test:mocha
+        run: yarn workspace ember-cli-fastboot test:mocha
```

Comment:
These commands are failing CI, not exactly sure why. Can you run it locally without error?

Reply:
I closed the PR and re opened the PR again and it is now working.

@rwjblue ^^